src/api/controllers/profileController.js

```diff
@@ -15,27 +15,43 @@ const { Article, User } = models;
  */
 export default class ProfileController {
   /**
-     *
-     * update current user's profile
-     * @static
-     * @param {*} req The request Object
-     * @param {*} res The response Object
-     * @memberof ProfileController
-     * @return {Object} The response Object
-     */
+   *
```

Comment:
@nkpremices can you clarify what this PR is for? The PR title and the changes made don't seem to match.

Reply:
`GET api/profiles/username` is the route that was there before. With it, users could get their own profile if they provide their `username`.

Now I faced a problem. 
After a successful login, I was able to get the username of the user from the store. But in case the user reloads the page, I was not able to get the username again cause the login actions won't be available in the store anymore.

Hence, I created a route that would return the profile of the user just with the token cause I could easily get it from the local storage.

That's what this PR is about

Reply:
okay, noted.